internal/server/server_test.go

```diff
@@ -278,20 +278,30 @@ func TestComposeImage(t *testing.T) {
 			Distribution:   "fedora-33",
 			ImageRequests: []ImageRequest{
 				ImageRequest{
-					Architecture:  "x86_64",
-					ImageType:     "qcow2",
-					UploadRequest: UploadRequest{},
+					Architecture: "x86_64",
+					ImageType:    "qcow2",
+					UploadRequest: UploadRequest{
+						Type: "aws",
+						Options: AWSUploadRequestOptions{
+							ShareWithAccounts: []string{"test-account"},
```

Comment:
ah the accounts here are 12-digit numbers, so like `012345678912`, not sure if we can somehow force that format in the openapi spec though